src/Game.java

```diff
@@ -40,10 +37,15 @@ public static void main(String[] args) {
         Keyboard.close();
     }
 
-    private static void TravelLoop(Player player, Landmark destination){
+    private static void TravelLoop(Player player, Node current, Node destination, Destination finalDestination){
+        //int startingMilesTraveled = player.getMilesTraveled();  Not used YET
+        int currentTravelMiles = 0;
+        int distanceToDestination = finalDestination.getDistance();
         int travelDelay = Settings.getInt("travel_delay");
         int travelDistance = Settings.getInt("travel_distance");
-        while(player.getMilesTraveled() < destination.distance){
+        while(currentTravelMiles < distanceToDestination) {
+                //destination.p1Miles + startingMilesTraveled)){
+
             //travel the road
             Utils.println("travel", travelDistance);
```

Comment:
The `travelDistance` should not go over the remaining distance. For example, if `2 miles` are remaining to the destination then it should output `2 miles` instead of the `travelDistance`.

Reply:
Fixed